### PR TITLE
Iterable doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

New doc for Iterable source integration

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by ItsSudip — APPROVED** [review]
LGTM

File: stream-sources/iterable.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+| `email`  | `context.traits.email`    |
+| `userId`        | `userId`        |
+| `dataFields`    | `properties`  |
+| `eventName`     | `event`   |
+
+<div class="infoBlock">
+
+The `userId` is a unique identifier in the output payload. If not provided, Rudderstack creates a unique `userId` using `email`.
+</div>
+
```
Comment (utsabc) [inline stream-sources/iterable.mdx:52]: Can we mention the list of events supported from iterable to rudderstack?
@ItsSudip @rohithkaza

Comment (rohithkaza) [reply stream-sources/iterable.mdx:52]: Yeah, we can mention if needed,
These are the events that Iterable supports:

[Blast Send](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#blast-send)
[Email Bounce](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-bounce)
[Email Click](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-click)
[Email Complaint](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-complaint)
[Email Open](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-open)
[Email Send Skip](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-send-skip)
[Email Subscribe](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-subscribe)
[Email Unsubscribe](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#email-unsubscribe)
[Hosted Unsubscribe Click](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#hosted-unsubscribe-click)
[In-App Click](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#in-app-click)
[In-App Open](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#in-app-open)
[In-App Send](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#in-app-send)
[In-App Send Skip](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#in-app-send)
[Push Bounce](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#push-bounce)
[Push Open](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#push-open)
[Push Send](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#push-send)
[Push Send Skip](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#push-send-skip)
[Push Uninstall](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#push-uninstall)
[SMS Bounce](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#sms-bounce)
[SMS Click](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#sms-click)
[SMS Send](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#sms-send)
[SMS Received](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#sms-received)
[SMS Send Skip](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#sms-send-skip)
[Triggered Send](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#triggered-send)
[Web Push Send](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#web-push-send)
[Web Push Send Skip](https://support.iterable.com/hc/en-us/articles/208013936-System-Webhooks-#web-push-send-skip)

**Review by rohithkaza — APPROVED** [review]

File: stream-sources/iterable.mdx (hunk 1 above)
Comment (ishachopra0401) [reply stream-sources/iterable.mdx:52]: done

**Review by utsabc — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**